README.md

```diff
@@ -20,9 +20,7 @@ WOLMO is a an opinionated framework designed to reduce the time taken to develop
 WOLMO is made up of several modules, each one supporting a different role in the production of modern apps. The following are the current modules available for the WOLMO framework.
 
 * **CORE:** This module provides standalone base classes and utilities of the framework. As a general rule, this module will be required in any WOLMO-based project. CORE is also a generally required dependency of other modules.
```

Comment:
I would delete the `CORE is also a generally required dependency of other modules.` part, since Wolmo Core is not a required dependency for other modules.